web-components/src/components/modal/TxBuySuccessfulModal/TxBuySuccessfulModal.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { cn } from '../../../utils/styles/cn';
+import RegenStepper from '../../stepper';
+import { TxModal, TxModalProps } from '../TxModal';
+
+export interface TxBuySuccessfulModalProps extends TxModalProps {
+  title: string;
+  description?: string;
+  steps: string[];
+  bgClassName: string;
+}
+
+const TxBuySuccessfulModal: React.FC<TxBuySuccessfulModalProps> = ({
+  steps,
+  bgClassName,
+  ...props
+}) => {
+  return (
+    <TxModal
+      {...props}
+      classes={{
+        root: '!pt-0 !px-0',
+        description: 'text-sm',
+        content: 'px-5 sm:px-30 ',
+      }}
+      titleVariant="h5"
+      titleMobileVariant="h5"
+      header={
+        <RegenStepper
```

Comment:
yes but the labels depend on the payment option and types of credits (retired or not), check `getSteps` so I thought it made sense to use RegenStepper
also if we were to use an svg it wouldn't look totally the same on mobile/desktop because of scaling